tests/gradients/parameter_shift/test_parameter_shift.py

```diff
@@ -3209,18 +3211,19 @@ def test_tf(self, tol, broadcast):
             with pytest.raises(
                 NotImplementedError, match="Broadcasting with multiple measurements"
             ):
-                with tf.GradientTape() as t:
-                    jac = self.cost_fn(weights, coeffs1, coeffs2, dev, broadcast)
+                with tf.GradientTape() as _:
+                    _ = self.cost_fn(weights, coeffs1, coeffs2, dev, broadcast)
```

Comment:
Same as everywhere else.
```suggestion
                with tf.GradientTape():
                    self.cost_fn(weights, coeffs1, coeffs2, dev, broadcast)
```